server/server/src/cold_chain/temperature_breach.rs

```diff
@@ -126,8 +126,8 @@ fn upsert_temperature_breach(
             breach.start_unix_timestamp
         ))?;
 
-    let duration_milliseconds = match breach.end_unix_timestamp {
-        Some(end_unix_timestamp) => (end_unix_timestamp - breach.start_unix_timestamp)
+    let duration_milliseconds: i32 = match breach.end_unix_timestamp {
+        Some(end_unix_timestamp) => ((end_unix_timestamp - breach.start_unix_timestamp) * 1000)
```

Comment:
the value which comes from the CCA is in seconds 🙄

Reply:
In this case, do we need duration field in schema (we would for filtering ?) ? Or does it have to be in milliseconds.. we need to check that fridge tag integration is correct

Reply:
actually, we don't need duration in the schema. might just take that out.